applications/nrf_desktop/src/modules/Kconfig.ble

```diff
@@ -215,4 +215,32 @@ source "subsys/logging/Kconfig.template.log_config"
 
 endif
 
+comment "BLE QoS management"
+
+config DESKTOP_BLE_QOS_ENABLE
+	bool "Enable BLE channel map management"
+	default n
```

Comment:
It is best-practice to omit "default n". This will result in the same behaviour.